core/src/main/java/org/axonframework/deadline/quartz/DeadlineJob.java

```diff
@@ -0,0 +1,133 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.deadline.quartz;
+
+import org.axonframework.common.transaction.Transaction;
+import org.axonframework.common.transaction.TransactionManager;
+import org.axonframework.deadline.DeadlineMessage;
+import org.axonframework.deadline.DeadlineContext;
+import org.axonframework.deadline.DeadlineTargetLoader;
+import org.axonframework.messaging.unitofwork.DefaultUnitOfWork;
+import org.quartz.Job;
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.SchedulerContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Quartz job which depicts handling of a scheduled deadline message. The {@link DeadlineMessage} and {@link
+ * DeadlineContext} are retrieved from the {@link JobExecutionContext}. Transaction manager and deadline target
+ * loader are fetched from {@link SchedulerContext}.
+ *
+ * @author Milan Savic
+ * @since 3.3
+ */
+public class DeadlineJob implements Job {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DeadlineJob.class);
+
+    /**
+     * The key under which the {@link TransactionManager} is stored within {@link SchedulerContext}.
+     */
+    public static final String TRANSACTION_MANAGER_KEY = TransactionManager.class.getName();
+
+    /**
+     * The key under which the {@link DeadlineTargetLoader} is stored within {@link SchedulerContext}.
+     */
+    public static final String DEADLINE_TARGET_LOADER_KEY = DeadlineTargetLoader.class.getName();
+
+    @Override
+    public void execute(JobExecutionContext context) throws JobExecutionException {
+        if (LOGGER.isDebugEnabled()) {
+            LOGGER.debug("Starting a deadline job");
+        }
+
+        JobDetail jobDetail = context.getJobDetail();
+        JobDataMap jobData = jobDetail.getJobDataMap();
+
+        try {
+            SchedulerContext schedulerContext = context.getScheduler().getContext();
+            DeadlineMessage deadlineMessage = DeadlineJobDataBinder.deadlineMessage(jobData);
+            DeadlineContext deadlineContext = DeadlineJobDataBinder.deadlineContext(jobData);
+
+            TransactionManager transactionManager = (TransactionManager) schedulerContext.get(TRANSACTION_MANAGER_KEY);
+            DeadlineTargetLoader deadlineTargetLoader = (DeadlineTargetLoader) schedulerContext.get(
+                    DEADLINE_TARGET_LOADER_KEY);
+
```

Comment:
Super nit: unintentional additional white space I assume.

Reply:
Autoformat :/. Will break the line differently.